### PR TITLE
deploy objecttypes on a single server

### PR DESCRIPTION
**Pull request by annashamray**
fixes https://github.com/maykinmedia/objects-api/issues/59

### CODE REVIEW

File: deployment/single-server/app.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    django_app_docker_image_name: maykinmedia/objecttypes-api
+    django_app_docker_replicas: 1
+    django_app_docker_port_start: 12000
+
+    django_app_docker_volumes:
+      - name: "{{ django_app_docker_name_prefix }}-media"
+        mount: "/app/media"
+        mode: "rw"
+        owner: 1000
+      - name: "{{ django_app_docker_name_prefix }}-private-media"
```
Comment (joeribekker) [inline deployment/single-server/app.yml:39]: can remove this whol volume block i think

File: deployment/single-server/app.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+
+# Invoke using:
+#
+#     ansible-playbook app.yml --become --ask-become-pass
+#
+
+- name: Catalogi Importer
```
Comment (joeribekker) [inline deployment/single-server/app.yml:8]:
```suggestion
- name: Objecttypes API
```

File: deployment/single-server/app.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    django_app_docker_db_host: TODO
+    django_app_docker_db_name: objecttypes
+    django_app_docker_db_username: objecttypes
+    django_app_docker_db_password: TODO
+
+    ## Application docker container
+    django_app_docker_version: latest
+    django_app_docker_image_name: maykinmedia/objecttypes-api
+    django_app_docker_replicas: 1
+    django_app_docker_port_start: 12000
```
Comment (joeribekker) [inline deployment/single-server/app.yml:32]:
```suggestion
    django_app_docker_port_start: 12001
```

I know it's pretty arbitrary but for testing purposes let's not use the same docker port for both the objects api and objecttypes api.

**Review by joeribekker — CHANGES_REQUESTED** [review]

File: deployment/single-server/app.yml (hunk 3 above)
Comment (annashamray) [reply deployment/single-server/app.yml]: Sure!

File: deployment/single-server/app.yml (hunk 1 above)
Comment (annashamray) [reply deployment/single-server/app.yml]: Sure, I'll remove the whole `django_app_docker_volumes` block

**Review by joeribekker — APPROVED** [review]

**Pull request merged**